File: 001.Two Sum/Jimmy Xiang/001_update.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        d = {}
        for i in range(len(nums)):
            find = target  - nums[i]
            if d.get(find, None) == None:
                d[nums[i]] = i
            else:
                return [d[find], i]

File: 001.Two Sum/Jimmy Xiang/001_update.go
func twoSum(nums []int, target int) []int {
    res := [2]int{-1,-1}
    
    for i, v := range nums {
        for j, o := range nums {
            if i < j && (target-v) == o {
                res[0]=i
                res[1]=j
                
                return res[:]
            }
        }
    }
    
    return res[:]
}
